### PR TITLE
create welcomeBox component

### PR DESCRIPTION
**Pull request by cristito08**
-Note: searchbox will be added in future commit

image

### CODE REVIEW

File: skillsearch-react/src/components/homePage/welcomeBox/welcomeBox.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+}
+.card-action .action-text{
+    font-size: 20px;
+    font-weight: 400;
+    color: #6c757d;
+    padding: 20px 0px;
+}
+.intersys-mini {
+    height: 40px;
+}
```
Comment (mmartinezintersys) [inline skillsearch-react/src/components/homePage/welcomeBox/welcomeBox.css]: Just to keep a convention :), add a new line at the end of the file

File: skillsearch-react/src/components/homePage/welcomeBox/welcomeBox.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+                    
+                </Card>
+            </Col>
+        </Row>
+        
+    )
+}
+}
+
+export default WelcomeBox;
```
Comment (mmartinezintersys) [inline skillsearch-react/src/components/homePage/welcomeBox/welcomeBox.js]: Same as in welcomeBox.css file, add a new line at the end of the file

File: skillsearch-react/src/components/homePage/welcomeBox/welcomeBox.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import React, {Component} from 'react';
+import {Card, Row, Col} from 'react-materialize';
+import './welcomeBox.css';
+// import miniLogo from './../../../../public/img/intersys_mini_logo';
```
[the remark this answers is not in the document]
Comment (cristito08) [reply skillsearch-react/src/components/homePage/welcomeBox/welcomeBox.js]: done

File: skillsearch-react/src/components/homePage/welcomeBox/welcomeBox.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+                    </div>]}
+                >
+                    
+                </Card>
+            </Col>
+        </Row>
+        
+    )
+}
+}
```
[the remark this answers is not in the document]
Comment (cristito08) [reply skillsearch-react/src/components/homePage/welcomeBox/welcomeBox.js]: done

File: skillsearch-react/src/components/homePage/welcomeBox/welcomeBox.css (hunk 1 above)
Comment (cristito08) [reply skillsearch-react/src/components/homePage/welcomeBox/welcomeBox.css]: done

File: skillsearch-react/src/components/homePage/welcomeBox/welcomeBox.js (hunk 2 above)
Comment (cristito08) [reply skillsearch-react/src/components/homePage/welcomeBox/welcomeBox.js]: done

**Pull request merged**